pkg/api/v1beta1/types.go

```diff
@@ -673,6 +673,8 @@ type Minion struct {
 	Status NodeStatus `json:"status,omitempty" description:"current status of node"`
 	// Labels for the node
 	Labels map[string]string `json:"labels,omitempty" description:"map of string keys and values that can be used to organize and categorize minions; labels of a minion assigned by the scheduler must match the scheduled pod's nodeSelector"`
+	// External ID of the node assigned by the cloud provider
```

Comment:
Given that we run in physical machine settings too, do we want something more generic than "cloud provider"?  Perhaps:

"External ID of the node assigned by some machine database (e.g. a cloud provider)"